client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -65,20 +60,15 @@ function ReaderSubscriptionListItem( {
 				/>
 			</div>
 			<div className="reader-subscription-list-item__byline">
-				<span className="reader-subscription-list-item__site-title">{ <a href={ streamUrl }> { siteTitle } </a> }</span>
+				<span className="reader-subscription-list-item__site-title">
+					{ <a href={ streamUrl } className="reader-subscription-list-item__link"> { siteTitle } </a> }
+				</span>
+				<div>{ siteExcerpt }</div>
 				{ ! isEmpty( authorName ) &&
-					<span>
-						<span className="reader-subscription-list-item__by-text">
-							{ translate( 'by' ) }
-						</span>
-						<span><a href={ streamUrl }> { authorName } </a></span>
+					<span className="reader-subscription-list-item__by-text">
```

Comment:
`this.translate` is deprecated because it relied on mixins which is a feature going away in the next version of React.  Instead we are now using a fancy technique called Higher Order Components (HoC).

The last line in the file is:
`export default localize( ReaderSubscriptionListItem );`.

`localize` is a function that basically just returns the exact same component it was given, except it also gives it the `translate` function.  The only thing you need to do to make it work is to undelete line [45](https://github.com/Automattic/wp-calypso/pull/13011/files/ff8823a535a41a19116dae2a1d2b7addf72e3e0c#diff-d50296c534ba4b1085dba09fce69a8e6L45).

Then  you can use `translate` instead of `this.translate`